cumulus_library/actions/cleaner.py

```diff
@@ -31,7 +31,7 @@ def _execute_drop_queries(
 
 
 def get_unprotected_stats_view_table(
-    cursor: databases.DatabaseCursor,
+    config: base_utils.StudyConfig,
```

Comment:
this is unrelated - i am midstream in tinkering with this to get specific tables, and need access to the config so i can get at the schema object. But it's functionally equivalent, so I opted to not back it out as part of this PR.